pkg/cmd/start/start.go

```diff
@@ -81,28 +81,32 @@ func runGather(operator controller.GatherJob, cfg *controllercmd.ControllerComma
 		}
 		operator.Controller = cont
 
-		kubeConfigPath := cmd.Flags().Lookup("kubeconfig").Value.String()
-		if len(kubeConfigPath) == 0 {
-			klog.Fatalf("error: --kubeconfig is required")
-		}
-		kubeConfigBytes, err := ioutil.ReadFile(kubeConfigPath)
-		if err != nil {
-			klog.Fatal(err)
-		}
-		kubeConfig, err := clientcmd.NewClientConfigFromBytes(kubeConfigBytes)
-		if err != nil {
-			klog.Fatal(err)
-		}
-		clientConfig, err := kubeConfig.ClientConfig()
-		if err != nil {
-			klog.Fatal(err)
+		var kubeConfig *rest.Config
+		if kubeConfigPath := cmd.Flags().Lookup("kubeconfig").Value.String(); len(kubeConfigPath) > 0 {
```

Comment:
If the `kubeConfigPath` is a string would be better to check it by: `kubeConfigPath != ""`